4-projects/business_unit_1/dev/example_restricted_shared_vpc_project.tf

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  prefix         = "${var.env_code}_shared_restricted"
+  perimeter_name = "sp_${local.prefix}_default_perimeter_1234"
```

Comment:
If this doesn't need to be dynamically derived I think we can also remove this along with the env code variable

Reply:
yeah `perimeter_name` will need to be dynamically derived for testing

Reply:
I now have a variable for this. I have also wired up the IT for 4-projects.